packages/datacommons-mcp/.env.sample

```diff
@@ -0,0 +1,68 @@
+# Data Commons MCP Server Configuration
+# 
+# This file contains all available configuration options for the Data Commons MCP server.
+# To use this configuration:
+# 1. Copy this file to .env: cp .env.sample .env
+# 2. Edit .env with your actual values
+# 3. Uncomment and set the variables you need
+#
+# Environment variables take precedence over .env file values.
+# All DC_* variables are optional except DC_API_KEY and DC_TYPE.
+
+# =============================================================================
+# REQUIRED CONFIGURATION
+# =============================================================================
+
+# API key for Data Commons (required for all configurations)
+DC_API_KEY=your_datacommons_api_key_here
+
+# Type of Data Commons to use (optional, defaults to "base")
+# Options: "base" (production Data Commons) or "custom" (custom instance)
+DC_TYPE=base
+
+# =============================================================================
+# CUSTOM DC CONFIGURATION (required when # DC_TYPE=custom)
+# =============================================================================
+
+# Base URL for custom Data Commons instance (required when # DC_TYPE=custom)
+# Example: https://staging-datacommons-web-service-650536812276.northamerica-northeast1.run.app
+# DC_BASE_URL=https://staging-datacommons-web-service-650536812276.northamerica-northeast1.run.app
```

Comment:
super nit: I think it's a little confusing to use the term "base" in two different ways: one being base vs custom dc and then the second being the base url **for** custom dc. Could we rename this to something like `DC_URL` or `DC_ROOT_URL` to avoid this?